pyiron_continuum/damask/damaskjob.py

```diff
@@ -41,15 +83,99 @@ def loading(self, path=None):
         self._loading = self.input.loading.read(path)
     
     def loading_write(self):
-        self.input.loading.write('tensionX.yaml')
+        self.input.loading.write('loading.yaml')
         
     def material_write(self):
         self.input.material.write('material.yaml')
-    
+   
+    def generategeometryyaml(self,size=1.0e-5,grains=4,grids=16):
+        self.input.geometry['size']=size
+        self.input.geometry['grains']=grains
+        self.input.geometry['grids']=grids
+        self.input.geometry['vtifile']=''
+        size=np.ones(3)*size
+        cells=[grids,grids,grids]
+        N_grains=grains
+
+        # for voronoi seed
+        seeds=damask.seeds.from_random(size,N_grains,cells)
+        new_geom=damask.Grid.from_Voronoi_tessellation(cells,size,seeds)
+        # save our geometry file to a vti file
+        self.input.geometry['vtifile']=f'Polycystal_{N_grains}_{cells[0]}x{cells[1]}x{cells[2]}.vti'
+
+        new_geom.save(f'Polycystal_{N_grains}_{cells[0]}x{cells[1]}x{cells[2]}')
+        
+        self.executable="DAMASK_grid --geom %s --load loading.yaml"%(self.input.geometry['vtifile'])
+
+    def generateloadyaml(self,loadfilename='tensile.yaml',loadtype='tensile',time=1.0,step=10,interval=2):
+        self.input.loading['loadfile']=loadfilename
+        self.input.loading['time']=time
+        self.input.loading['step']=step
+        self.input.loading['interval']=interval
+        load_case = damask.Config(solver={'mechanical':'spectral_basic'},
+                          loadstep=[])
+        if loadtype=='tensile':
+            dotF = [1.0e-2,0,0, 0,'x',0,  0,0,'x']
+            P = ['x' if i != 'x' else 0 for i in dotF]
+
+            load_case['loadstep'].append({'boundary_conditions':{},
+                'discretization':{'t':time,'N':step},'f_out':interval})
+
+            load_case['loadstep'][0]['boundary_conditions']['mechanical'] = \
+                    {'dot_F':[dotF[0:3],dotF[3:6],dotF[6:9]],
+                     'P':[P[0:3],P[3:6],P[6:9]]}
+
+        load_case.save(loadfilename)
+        
+        self.executable="DAMASK_grid --geom %s --load %s"%(self.input.geometry['vtifile'],self.input.loading['loadfile'])
+
+
+    def SaveResult(self):
```

Comment:
You should add `SaveResult` to the `collect_output` function. This way, the results are collected automatically after job is run.